core/lcp/credential.py

```diff
@@ -31,6 +70,9 @@ def _get_or_create_persistent_token(
         :param patron: Patron object
         :type patron: core.model.patron.Patron
 
+        :param commit: True if a generated identifier should be committed to the database
+        :type commit: bool
```

Comment:
Minor: no need for docstring `type` for params with type hints in the signature.

Reply:
If less is more then think how much more more would be! :smile: